src/index.js

```diff
@@ -9,6 +9,7 @@ import { BrowserRouter } from 'react-router-dom';
 import App from './base/App';
 import configureStore from './store/configureStore';
 import './index.scss';
+import './resources/font';
```

Comment:
Should this be in this PR?

Reply:
I figured to include it since it was missing. Should it have it's own PR?

Reply:
yea lets give it its own PR